service/matching/matching_engine_test.go

```diff
@@ -2836,7 +3169,7 @@ func getKey(dbq *PhysicalTaskQueueKey) dbTaskQueueKey {
 }
 
 func newTestTaskManager(logger log.Logger) *testTaskManager {
-	return &testTaskManager{queues: make(map[dbTaskQueueKey]*testPhysicalTaskQueueManager), logger: logger}
+	return &testTaskManager{queues: make(map[dbTaskQueueKey]*testPhysicalTaskQueueManager), logger: logger, dbConditionalFailedError: false, dbServiceError: false}
```

Comment:
you don't need to set these to false explicitly

Reply:
yeah, I don't (even for the above comment) but thought it was better to be as explicit as possible for the reader. I can remove them though, thanks!